README.md

```diff
@@ -1,3 +1,10 @@
+
+# Personal Portfolio
+
+# fed1k.github.io
+
```

Comment:
I suggest that you instead use a more in-depth README file for your project.

- [ ] Please checkout [https://github.com/microverseinc/readme-template](https://github.com/microverseinc/readme-template), for a better understanding of how to set up a ReadME file.
- [ ] You are also required to include the link of the online version of your portfolio in the README after deploying it using github pages.